Sources/PackageModel/Platform.swift

```diff
@@ -55,6 +55,7 @@ public struct Platform: Equatable, Hashable {
     public static let tvOS: Platform = Platform(name: "tvos", oldestSupportedVersion: "9.0")
     public static let watchOS: Platform = Platform(name: "watchos", oldestSupportedVersion: "2.0")
     public static let linux: Platform = Platform(name: "linux", oldestSupportedVersion: .unknown)
+    public static let android: Platform = Platform(name: "android", oldestSupportedVersion: "7.0")
```

Comment:
[The Termux package repo supports back to 7.0](https://github.com/termux/termux-packages/blob/master/README.md#project-structure) and three years back seemed a reasonable default, but I don't care what version is used here.

Reply:
I'm a little concerned about committing to a versioning scheme here.

First, the backwards compatibility story works a bit differently on Android, where there are two versions to worry about at build time: `minSdkVersion` (roughly equivalent to the Apple "deployment target" concept), and `targetSdkVersion` (roughly equivalent to the Apple SDK version, but on Android you CAN change this, on Apple platforms it's fixed by a particular Xcode version). There's also maxSdkVersion, but this is deprecated and ignored nowadays. See https://developer.android.com/guide/topics/manifest/uses-sdk-element.html#target for more info.

Secondly, developers don't use the Android marketing versions like "7.0", "8.0", etc., for anything. They use [API levels](https://source.android.com/setup/start/build-numbers) instead, and no one has decided yet how Swift availability will work on Android (whether it would use API levels like native Android tools, or whether it would attempt to map marketing versions to API levels).

Let's leave this at "unknown" until we have a better idea of how this should work in the Swift world.